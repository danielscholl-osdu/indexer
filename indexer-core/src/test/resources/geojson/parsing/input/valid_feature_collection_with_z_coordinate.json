{
    "type": "FeatureCollection",
    "bbox": null,
    "features": [
        {
            "type": "Feature",
            "bbox": null,
            "geometry": {
                "type": "Point",
                "bbox": null,
                "coordinates": [
                    5.7500000010406245,
                    59.000000000399105,
                    1.0
                ]
            },
            "properties": {}
        }
    ],
    "properties": {},
    "persistableReferenceCrs": null,
    "persistableReferenceUnitZ": "{\"scaleOffset\":{\"scale\":1.0,\"offset\":0.0},\"symbol\":\"m\",\"baseMeasurement\":{\"ancestry\":\"Length\",\"type\":\"UM\"},\"type\":\"USO\"}"
}