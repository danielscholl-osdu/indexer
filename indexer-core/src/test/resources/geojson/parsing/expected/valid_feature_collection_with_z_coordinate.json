{
    "type": "geometrycollection",
    "geometries": [
        {
            "type": "point",
            "coordinates": [
                5.7500000010406245,
                59.000000000399105,
                1.0
            ]
        }
    ]
}